### PR TITLE
CCITCARBON-310 Add the var-file declared by user as an extra_vars in the ansible orchestrate and execute task

### PR DESCRIPTION
**Pull request by rujutashinde**

### CODE REVIEW

File: tests/localhost_scenario/scenario_local.yml [hunk 1]
```diff
@@ -113,9 +113,9 @@ orchestrate:
     hosts: localhost
     ansible_playbook:
       name: ansible/template_host_test_playbook_tasks.yml
-    ansible_options:
-      extra_vars:
-        file: variable.yml
+#    ansible_options:
```
Comment (JunqiZhang0) [inline tests/localhost_scenario/scenario_local.yml:116]: Do we need these commented lines?

Comment (rujutashinde) [reply tests/localhost_scenario/scenario_local.yml]: removed

**Review by JunqiZhang0 — APPROVED** [review]
LGTM

**Comment by shay6** [comment]
should we add an explanation in the docs what is the precedence when the var files are collected? (over cli, teflo.cfg, var_file.yml. vars folder)
like it's explained in the ticket

**Review by shay6 — APPROVED** [review]

**Pull request merged**